### PR TITLE
archive-ops-74 Archive issue discovered during thumbnail generation

### PR DESCRIPTION
**Pull request by TBRC-JimK**
This change addresses a part of the fix for this issue: there are times when we need to restructure a work's published tree - Move things around, delete objects, etc. The existing implementation had no framework for doing that. This change creates entries in the config file for independently replacing either the web or the archive artifact.

Test collateral: see sattva:~(me)/prod/archive-ops-74/archive-ops-74.config

### CODE REVIEW

File: scripts/processing/syncAnywhere/processWeb.sh [hunk 1]
```diff
@@ -20,6 +20,7 @@ existDBIGUpdaterURL="http://www.tbrc.org:51173/exist/rest/db/modules/admin/image
     function usage() {
         echo $SYNOPSIS
         echo "    where -h  prints this message"
+        echo "    -W  replaces the entire tree, deleting anything not found on the source tree"
```
Comment (TBRC-Travis) [inline scripts/processing/syncAnywhere/processWeb.sh:23]: Suggest to update to "deleting anything in the destination not found in the source tree"

File: scripts/processing/syncAnywhere/syncOneWork.sh [hunk 2]
```diff
@@ -107,14 +108,16 @@ function processArchive() {
 
     dest=${1:?"Bad arg to ${FUNCNAME[0]}. Requires destinaton path"}
     testWork=${workName:?"workName required definition for ${FUNCNAME[0]}"}
+    rsArgs="${arcReplace} --exclude \".DS_Store\" --exclude \"Thumbs.db\""
```
Comment (TBRC-Travis) [inline scripts/processing/syncAnywhere/syncOneWork.sh:111]: Nice

File: scripts/processing/syncAnywhere/processWeb.sh (hunk 1 above)
Comment (TBRC-JimK) [reply scripts/processing/syncAnywhere/processWeb.sh:23]: Done, mostly

**Comment by TBRC-JimK** [comment]
Resolves #74

**Pull request merged**